brands/amenity/restaurant.json

```diff
@@ -1791,6 +1791,17 @@
       "cuisine": "pizza",
       "name": "Pieology Pizzeria"
     }
+  },  
+  "amenity/restaurant|Pizza Delight": {
+    "countryCodes": ["CA"],
```

Comment:
This is where TRAVIS seemed to have an issue @bhousel . Is the country code supposed to be lower-case for this to be fixed?

Reply:
Yes it should be lower case to validate